projects/plugins/jetpack/modules/widgets/blog-stats.php

```diff
@@ -143,7 +143,9 @@ function widget( $args, $instance ) {
 		// Get the Site Stats.
 		$views = $this->get_stats();
 
-		if ( ! empty( $views ) ) {
+		if ( 0 === $views ) {
+			esc_html_e( 'There is nothing to display yet', 'jetpack' );
```

Comment:
I like the idea, that's a nice addition!

As for the wording, I'll let the experts weigh in, but if we need to improve it, maybe we should use something similar to the Top Posts widget?
https://github.com/Automattic/jetpack/blob/e13487d0dbbe4c743739b6f1a3e37c143a8c2614/projects/plugins/jetpack/modules/widgets/top-posts.php#L311

Reply:
That's a good idea for when the current user is an admin! Nice